src/tools/compiletest/src/runtest.rs

```diff
@@ -1578,8 +1610,7 @@ impl<'test> TestCx<'test> {
         let newpath = env::join_paths(&path).unwrap();
         command.env(dylib_env_var(), newpath);
 
-        let mut child = command
-            .spawn()
+        let mut child = disable_error_reporting(|| command.spawn())
```

Comment:
Doesn't this mean that we're only executing one test at a time since this is globally serialized?

Reply:
No. `spawn` doesn't wait for the completion of the process.

Reply:
Ah oops yes, indeed!